### ISSUE TITLE
in gilneas when complete this quest the server crash?!

### ISSUE THREAD

**Issue by cnmir2** [Arkania/ArkCORE-NG#197]
in gilneas when complete this quest the server crash?!

quest   [The Rebel Lord's Arsenal]

![1](https://cloud.githubusercontent.com/assets/14962131/20424281/fd415f44-adaf-11e6-80ba-b1ff092d03f4.png)
![2](https://cloud.githubusercontent.com/assets/14962131/20424294/0b93346e-adb0-11e6-8418-6ef4f1d3749e.png)

**Comment by cnmir2** [comment]
test this crash maybe the database problem the 1.4.1 (x64) repack   have not this problem!

but frank i  use  your 1.4.1 (x64) repack  create a undead hunter  do some quest sudendly  the server crash!

**Comment by gpn39f** [comment]
this is not a crash ???!!!!!
click on the message box:  Quest Discovered! (in the box is the text: click to view quest)
then you receive next quest box and you can accept the quest and finish the quest..

**Comment by cnmir2** [comment]
yes !but frank i use your 1.4.1 (x64) repack create a undead hunter do some quest sudendly the server crash!  now i  am going to change the mmap vmap map and dbc from your old forum! try to test again! then give your feedback

**Comment by cnmir2** [comment]
also crash ! i use one undead hunter character  and another accunt undead priest to join in the world! do some quest kill monster  then the sever crash!

i use your old forum a.zip b.zip c.zip d.zip (map vmap mmap dbc file)
![abcd](https://cloud.githubusercontent.com/assets/14962131/20455729/ba62a676-ae9d-11e6-9a01-de57c8d871b8.png)

**Comment by cnmir2** [comment]
frank  could you have private server ??if you have give me your ip i want to join and  test your server online!

**Comment by cnmir2** [comment]
frank sent me  another one more account please!

**Comment by cnmir2** [comment]
crash|???

**Comment by anubislinux** [comment]
To me it happens the same thing, when I will finish the quest in the winery of the server closes, in the server log shows the following:

Excuse my English, it is not my language

No defined handler for opcode [CMSG_MOVE_WATER_WALK_ACK 0x3B00 (15104)] sent by [Player:

**Comment by anubislinux** [comment]
To me it happens the same thing, when I will finish the quest in the winery of the server closes, in the server log shows the following:

Excuse my English, it is not my language

No defined handler for opcode [CMSG_MOVE_WATER_WALK_ACK 0x3B00 (15104)] sent by [Player:

**Comment by gpn39f** [comment]
@anubislinux pls give me more info..
quest link to wowhead
data to youre character : race and class
what is the last action you made.. 
and more data to the crash..

some crash data are written to the console window. 
you have to take picture from linux-desktop , with bigsizeed  worldserver console

**Comment by anubislinux** [comment]
With any class that starts, when I reach the quest [Rebel Lord's Arsenal] that I will complete in the basement where Josiah Aver closes the core, in the server log what it shows is:
No defined handler for opcode [CMSG_MOVE_WATER_WALK_ACK 0x3B00 (15104)] sent by [Player: Anubis (Guid: 3, Account: 1)]
No defined handler for opcode [CMSG_MOVE_WATER_WALK_ACK 0x3B00 (15104)] sent by [Player: Anubis (Guid: 3, Account: 1)]
Halting process...
When I reload the core that I access gilneas is empty, and does not let me access the basement again, another thing to know Josiah Avery has to bite me at that time and neither does.
If I activate GM ON I get everyone in Gilneas but nothing is activated.
Apparently it must be when Josiah Aver has to bite
I'm using:
ArkDB 7.9.19 - 2016-OCT-01 for ArkCORE4 NG
With updates until November 13

**Comment by cnmir2** [comment]
@anubislinux  could you have server crash in undead start area ?

**Comment by anubislinux** [comment]
@cnmir2 yes

**Comment by Desarrollo-zeros** [comment]
![image](https://cloud.githubusercontent.com/assets/23331674/20460227/b17254e6-aeaa-11e6-8ab3-607a674e5723.png)
![image](https://cloud.githubusercontent.com/assets/23331674/20460230/bbdc4d24-aeaa-11e6-90a6-164ed1e4dbec.png)
![image](https://cloud.githubusercontent.com/assets/23331674/20460231/c08118be-aeaa-11e6-9039-eff0638884d1.png)

**Comment by Desarrollo-zeros** [comment]
@gpn39f Bug zone in undead dark grave

![image](https://cloud.githubusercontent.com/assets/23331674/20460244/476490cc-aeab-11e6-9803-51b58364fa36.png)
![image](https://cloud.githubusercontent.com/assets/23331674/20460247/5fba8906-aeab-11e6-8e44-29857478430b.png)
![image](https://cloud.githubusercontent.com/assets/23331674/20460249/6a084d08-aeab-11e6-84fe-b7bbe8ba8f5d.png)
![image](https://cloud.githubusercontent.com/assets/23331674/20460254/884970a8-aeab-11e6-8001-e52ac3b841f8.png)
![image](https://cloud.githubusercontent.com/assets/23331674/20460268/f563bb9e-aeab-11e6-9792-0475a904bad0.png)
![image](https://cloud.githubusercontent.com/assets/23331674/20460273/091731a2-aeac-11e6-95e7-e9594f985b22.png)
![image](https://cloud.githubusercontent.com/assets/23331674/20460277/2e6e23c0-aeac-11e6-9650-6c95f1c65530.png)

**Comment by Desarrollo-zeros** [comment]
> Test --> Hunter --> DIE
> Test --> priest  ---> DIE
> Test --> Other races well, for an instant falls after a few seconds, with priest and hunters is instantly floor that area ---> DIE
> Command: .gps [Player: Testsxx (Guid: 9) (Account: 7) X: 1659.233154 Y: 1678.132568 Z: 121.024818 O: 3.181723 Map: 0 (Reinos del Este) Area: 2117 (Sepulcro Sombrío) Zone: 85 (Claros de Tirisfal)]

**Comment by Desarrollo-zeros** [comment]
[Target: King Genn Greymane (Guid: 27676968 Entry: 35112 DBGuid: 250817)]
die Core  --> http://www.wowhead.com/g00/quest=14159/the-rebel-lords-arsenal <--
At the moment of delivering and receiving reward of the mission the core dies

![image](https://cloud.githubusercontent.com/assets/23331674/20460403/e3fa5946-aeae-11e6-9be8-a4ac1b33ed08.png)
![image](https://cloud.githubusercontent.com/assets/23331674/20460404/e772a402-aeae-11e6-8803-2a7e252ca64d.png)
![image](https://cloud.githubusercontent.com/assets/23331674/20460405/f0ed9d0c-aeae-11e6-9130-d662f1b96f1a.png)

**Comment by gpn39f** [comment]
@Desarrollo-zeros  why is youre server need 3 min 20sec to load up..
a normal windows computer with 8gb ram 2 GHz cpu should take below 30 sec..

and why are you compiling 32 bit cpu if youre on win 10..

so: i think first failure you made is in cmake.. here you answer 
on FIRST question: what compiler you have: you choose is "vs2015 win64"
in RED window you have NOT to change anything.. 
redo "configure" a second time, window is white
then "generate"

after starting vs2015 you go in the top of VS
choose there release 64 bit..
and compile

finish.. result in \bin\release..

**Comment by Desarrollo-zeros** [comment]
In linux same does the same, frank gives death to the server I have debian 8 x64

**Comment by Desarrollo-zeros** [comment]
In my laptop also does the same as on my desktop pc

![image](https://cloud.githubusercontent.com/assets/23331674/20463347/806be0ac-aefe-11e6-82df-362173428b73.png)
![image](https://cloud.githubusercontent.com/assets/23331674/20463365/caa20228-aefe-11e6-8439-ba0c44d441de.png)

**Comment by Desarrollo-zeros** [comment]
dudo que la falla sea por ello debe ser algo de core ![image](https://cloud.githubusercontent.com/assets/23331674/20463417/5c627dcc-af00-11e6-9449-1542a24d0b8c.png)

**Comment by Desarrollo-zeros** [comment]
@gpn39f  --> It takes 3 minutes and more because I was running a database that weighs around 2GB by means of cosola, then the memory resources were being spent, but normally in windows it lasts 10 to 25 seconds in execution to start server and in linux Muho less, but that is not the problem, look at the photos well, I did test in linux, and in my other pc, and nothing like that dies in that part the server,
![image](https://cloud.githubusercontent.com/assets/23331674/20463484/354871ea-af02-11e6-9f38-3895fa99b2fd.png)

**Comment by gpn39f** [comment]
pls. write me pm in our forum

**Comment by Desarrollo-zeros** [comment]
![image](https://cloud.githubusercontent.com/assets/23331674/20465599/a6d1e4aa-af2e-11e6-8265-77009292ae88.png)

**Comment by ghost** [comment]
Can confirm, when talking to this guy the server hangs out and you cannot do anything until restarting it.

**Comment by ghost** [comment]
It seems that this crashing at the undead starting zone is random. I do not think its anything to do with the quest 28608 as when I sat at the starting zone, the server crashed.
